Source/ModCompendiumLibrary/ModSystem/Builders/Persona5ModCpkModBuilder.cs

```diff
@@ -69,7 +69,7 @@ public VirtualFileSystemEntry Build( VirtualDirectory root, string hostOutputPat
             Log.Builder.Info( "Building mod.cpk" );
             var cpkModCompiler = new CpkModBuilder();
             var cpkFilePath = hostOutputPath != null ? Path.Combine( hostOutputPath, "mod.cpk" ) : null;
-            var cpkFile = cpkModCompiler.Build( modFilesDirectory, cpkFilePath );
+            var cpkFile = cpkModCompiler.Build( modFilesDirectory, cpkFilePath, string.Empty);
```

Comment:
You're passing string.Empty instead of the compression parameter, why can't mod.cpks be compressed too?